litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -221,3 +227,27 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 		IsCustomWorkflow:    input.IsCustomWorkflow,
 	}, nil
 }
+
+func DeleteWorkflow(workflow_id string, r store.StateData) (bool, error) {
+
+	workflows, err := database.GetWorkflowsByIDs([]*string{&workflow_id})
+	if err != nil {
+		return false, err
+	}
+
+	bool, err := database.DeleteChaosWorkflow(workflow_id)
```

Comment:
The workflow runs are also deleted in this one right?

Reply:
yes, it is already present. Non-removal of workflow runs after deletion of workflow is coming in separate PR(which requires some changes in the list API).